aunt-leahs-app/src/components/SelectBox.jsx

```diff
@@ -1,44 +1,37 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { InputLabel, MenuItem, FormControl, Select } from '@material-ui/core';
 
-class SelectBox extends React.Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-            items: this.props.items || [],
-            selectedItem: ''
-        };
-        this.handleSelect = this.handleSelect.bind(this);
-    }
-    handleSelect = (e) => {
-        this.props.onSelectItem(e.target.value)
-        this.setState({ selectedItem: e.target.value });
+const SelectBox = ({ name, items, onSelectItem }) => {
+
+    const [selectedItem, setSelectedItem] = useState('');
+
+    const handleChange = event => {
+        onSelectItem(event.target.value);
+        setSelectedItem(event.target.value);
     }
 
-    render() {
-        return (
-            <div>
-                <FormControl fullWidth={true}>
-                    <InputLabel id='volunteer-select-label'>{this.props.name || 'Select'}</InputLabel>
-                    <Select
-                        labelId='volunteer-select-label'
-                        id='my-select'
-                        value={this.state.selectedItem}
-                        onChange={this.handleSelect}
-                    >
-                        <MenuItem value=''>
-                            <em>None</em>
+    return (
+        <div>
+            <FormControl fullWidth={true}>
+                <InputLabel id='volunteer-select-label'>{name || 'Select'}</InputLabel>
```

Comment:
I'm not certain but I believe since there is no default value for `name` (i.e. `''`), trying to access it should yield null or undefined. So I think in this case we should be using `??` instead of `||`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

Reply:
Also, just like the other comment, I think the id needs to be unique and this is a generic component (so it really shouldn't specify "volunteer" either... 🤔)